benchmark/src/DriverBench/ParallelMultiFileImportBench.php

```diff
@@ -182,17 +190,20 @@ public static function provideProcessesParameter(): Generator
      *  - fgets
      *  - stream_get_line 🏆
      */
-    public static function importFile(string $file): void
+    public static function importFile(string|array $files): void
     {
         $namespace = sprintf('%s.%s', Utils::getDatabaseName(), Utils::getCollectionName());
 
         $bulkWrite = new BulkWrite();
-        $fh = fopen($file, 'r');
-        while (($line = stream_get_line($fh, 10_000, "\n")) !== false) {
-            $bulkWrite->insert(Document::fromJSON($line));
+        foreach ((array) $files as $file) {
+            $fh = fopen($file, 'r');
+            while (($line = stream_get_line($fh, 10_000, "\n")) !== false) {
+                $bulkWrite->insert(Document::fromJSON($line));
+            }
+
+            fclose($fh);
```

Comment:
Nit: this should be wrapped in a `finally` block to ensure the handle is closed. I'm not sure if the handle is closed automatically when the variable goes out of scope, but better safe than sorry.